### PR TITLE
Feature update route

### PR DESCRIPTION
**Pull request by acrankin**
update and getbyID both working

### CODE REVIEW

File: lib/routes/evenings.js [hunk 1]
```diff
@@ -14,4 +14,28 @@ module.exports = router
...
+
+    .put('/:id', ensureAuth, (req, res, next) => {
+        const { id } = req.params;
+        const { rating } = req.body;
+        Evening.findOneAndUpdate(
+            { _id: id },
+            { rating: rating },
+            { new: true, runValidators: true })
+            .populate({ path: 'logs', select: { __v: false } })
+            .select({ __v: false })
```
Comment (MackenzieBerliner-Glasser) [inline lib/routes/evenings.js]: you dont need this because `toJSON` does this.

File: lib/routes/evenings.js [hunk 2]
```diff
@@ -14,4 +14,28 @@ module.exports = router
...
+            .lean()
+            .then(evening => res.json(evening))
+            .catch(next);
+    })
+
+    .get('/:id', ensureAuth, (req, res, next) => {
+        const { id } = req.params;
+        Evening.findById(id)
+            .populate({ path: 'logs', select: { __v: false } })
+            .select({ __v: false })
```
Comment (MackenzieBerliner-Glasser) [inline lib/routes/evenings.js]: same as above

File: lib/routes/evenings.js (hunk 1 above)
Comment (acrankin) [reply lib/routes/evenings.js]: we need it because of `lean()`

File: lib/routes/evenings.js (hunk 2 above)
Comment (acrankin) [reply lib/routes/evenings.js]: same deal, we need it because of `lean()`

**Pull request merged**